ansible/roles/stack-sunbird/templates/search-service_logback.xml

```diff
@@ -0,0 +1,62 @@
+<configuration>
+
+	<conversionRule conversionWord="coloredLevel" converterClass="play.api.libs.logback.ColoredLevel" />
+
+	<!-- transaction-event-trigger START -->
+	<timestamp key="timestamp" datePattern="yyyy-MM-dd"/>
+	<!-- common transactions logs -->
+	<appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+		<encoder>
+			<pattern>%d %msg%n</pattern>
+		</encoder>
+	</appender>
+
+	<appender name="ASYNCSTDOUT" class="ch.qos.logback.classic.AsyncAppender">
+		<appender-ref ref="STDOUT" />
+	</appender>
+
+
+	<logger name="play" level="INFO" />
+	<logger name="DefaultPlatformLogger" level="INFO" />
+	<!-- Telemetry Loggers-->
+
+	<root level="INFO">
+		<appender-ref ref="ASYNCSTDOUT" />
+	</root>
+
+
+	<appender name="kafka-appender" class="com.github.danielwegener.logback.kafka.KafkaAppender">
+		<encoder class="ch.qos.logback.classic.encoder.PatternLayoutEncoder">
+			<pattern>%msg</pattern>
+		</encoder>
+
+		<topic>{{env_name}}.telemetry.raw</topic>
+		<!-- ensure that every message sent by the executing host is partitioned to the same partition strategy -->
+        <keyingStrategy class="com.github.danielwegener.logback.kafka.keying.HostNameKeyingStrategy" />
+        <!-- block the logging application thread if the kafka appender cannot keep up with sending the log messages -->
+        <deliveryStrategy class="com.github.danielwegener.logback.kafka.delivery.BlockingDeliveryStrategy">
+            <!-- wait indefinitely until the kafka producer was able to send the message -->
+            <timeout>0</timeout>
+        </deliveryStrategy>
+		
+		<!-- each <producerConfig> translates to regular kafka-client config (format: key=value) -->
+		<!-- producer configs are documented here: https://kafka.apache.org/documentation.html#newproducerconfigs -->
+		<!-- bootstrap.servers is the only mandatory producerConfig -->
+		<producerConfig>bootstrap.servers={{lp_kafka_url}}</producerConfig>
```

Comment:
What is this variable lp_kafka_url? Please define in the inventory in the template below

Reply:
lp_kafka_url -  is already present as part of content-service.